solution_template/jenkins/nested/VM-password.json

```diff
@@ -0,0 +1,147 @@
+﻿{
```

Comment:
I believe these nested templates can be used for other (Spinnaker) templates as well. Can you move them down a level, outside the "jenkins" folder?